apps/frontend/src/components/Auth/UserUtils.tsx

```diff
@@ -45,6 +46,6 @@ export const getCurrentUser = async function (): Promise<UserSchema> {
   const cognitoCurrentUser = await Auth.currentUserInfo();
   console.log(cognitoCurrentUser);
   const currUser = CognitoUser.parse(cognitoCurrentUser);
-  //TODO: Also make a call to apiClient.getUser() to get the group for this user
-  return convertCognitoUser(currUser);
+
+  return await apiClient.getUser(currUser.attributes.sub)
```

Comment:
Perfect! This is great.

Reply:
I think it would be worthwile to place this in to a try catch block in case the getUser fails. Can someone else comment on this?